resources/quality/weedo_x40/Extrudr/weedo_x40_0.4_Extrudr_GreenTECPro_Anthracit_Draft.inst.cfg

```diff
@@ -0,0 +1,28 @@
+[general]
+version = 4
+name = Draft
+definition = weedo_x40
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = draft
+weight = 0
+material = Extrudr_GreenTECPro_Anthracite_175
+variant = 0.4mm Nozzle
+
+[values]
+speed_print = 60
+layer_height_0 = =round(layer_height * 1.25, 2)
+material_print_temperature_layer_0 = =material_print_temperature + 10
+material_initial_print_temperature = =material_print_temperature - 5
+material_final_print_temperature = =material_print_temperature - 5
+material_bed_temperature = =default_material_bed_temperature
+material_bed_temperature_layer_0= =material_bed_temperature + 10
+material_flow_layer_0 = =material_flow + 4
+skirt_brim_material_flow= =material_flow + 4
+material_standby_temperature = 100
+support_z_distance = 0.2
```

Comment:
Why override this again? The definition sets it to  "layer_height if layer_height >= 0.16 else layer_height*2"